packages/react-native-editor/__device-tests__/gutenberg-editor-gallery.test.js

```diff
@@ -1,19 +0,0 @@
-/**
- * Internal dependencies
- */
-import { blockNames } from './pages/editor-page';
-
-describe( 'Gutenberg Editor Gallery Block tests', () => {
-	it( 'should be able to add a gallery block', async () => {
-		await editorPage.addNewBlock( blockNames.gallery );
-		await editorPage.driver.sleep( 1000 );
-		await editorPage.closePicker();
-
-		const galleryBlock = await editorPage.getBlockAtPosition(
-			blockNames.gallery
-		);
-
-		expect( galleryBlock ).toBeTruthy();
-		await editorPage.removeBlockAtPosition( blockNames.gallery );
-	} );
```

Comment:
This is already covered with this test case:
https://github.com/WordPress/gutenberg/blob/4a0c09aafcc7738cd4e25aea9e42a14b8cd3b3aa/packages/block-library/src/gallery/test/index.native.js#L58-L67